### ISSUE TITLE
Can not open torrent page

### ISSUE THREAD

**Issue by baterflyrity** [imDMG/qBt_SE#26]
Hi! Thanks for plugins, they are really great.

Unfortunately, when I click on search result and select "go to torrent description page" my browsers opens this repository web-page instead of real tracker page.

**Comment by imDMG** [comment]
Hello.
About what plug-in you talking?

**Comment by baterflyrity** [comment]
All/

**Comment by imDMG** [comment]
There is an error as result?

**Comment by baterflyrity** [comment]
Yes, I've figured out that this happens when tracker is blocked. Since using VPN solves this issue.

Unfortunately, while tracker is blocked the plugin gives such results:

- open download window - nothing happens
- download - nothing happens
- name - [<search request>][Error]: <url> is not response! Maybe it is blocked.
- download link - <tracker>/error
- description link - https://github.com/imDMG/qBt_SE
- size - 1,000 TB
- seeds - 100
- leechers - 100

Can it just show an error or ask to connect over VPN?

**Comment by imDMG** [comment]
In common case display of a real error don't give understanding of the issue. This will more confuse a regular user.

**Comment by baterflyrity** [comment]
In contrast opening github instead of tracker confuses much more than displaying any kind of error. Because users can can at least distinguish errors.
